src/lib/ngsiNotify/QueueNotifier.cpp

```diff
@@ -100,8 +101,8 @@ void QueueNotifier::sendNotifyContextRequest(NotifyContextRequest* ncr, const st
     spathList = "";
   }
 
-  ci.outFormat = format;
-  std::string payload = ncr->render(&ci, NotifyContext, "");
+  ci.outFormat = JSON;
```

Comment:
They are conceptually different things... I tend to prefer not mixing churras with merinas ;)